apps/web/src/ee/clerk/components/QuestionnaireForm.tsx

```diff
@@ -58,6 +59,20 @@ export function QuestionnaireForm() {
     };
     await updateOrganizationMutation(updateClerkOrgDto);
 
+    const hubspotContext = document.cookie
```

Comment:
You can use [js-cookies](https://github.com/novuhq/novu/blob/5cd0e5a6a1c9aaa0fd042b62959d78ae02c58994/apps/web/package.json#L99) to get the hubspot cookie value